pymmcore_plus/mda/_engine.py

```diff
@@ -265,7 +266,9 @@ def run(self, sequence: MDASequence) -> None:
                 self._mmc.snapImage()
                 img = self._mmc.getImage()
 
-                self._events.frameReady.emit(img, event)
+                with contextlib.suppress(EmitLoopError):
```

Comment:
@ianhi  ... this was added from https://github.com/tlambert03/psygnal/pull/115 ... is this the behavior you wanted here?